.kokoro/runintegrationtests.sh

```diff
@@ -10,9 +10,7 @@ maybe_upload_coverage() {
     return 0
   fi
   
-  ./createcoveragereport.sh
-  # Assume we've created the coverage file by this point. If we haven't, there should already have been an error.
-  codecov -f "coverage/coverage-filtered.xml" --flag $1 --c $KOKORO_GITHUB_COMMIT --b $KOKORO_BUILD_NUMBER
+  ./createcoveragereport.sh --flag $1 --c $KOKORO_GITHUB_COMMIT --b $KOKORO_BUILD_NUMBER
```

Comment:
If we make createcoveragereport.sh just succeed with a message of "no coverage detected" or similar when there's no coverage directory, we could remove this whole function, simplifying things quite a bit.

Reply:
Oh but what we are checking for here is the codecov upload token not the coverage directory. The token won't be present when running from AppVeyor so we can't move the check for the token to createcoveragereport.sh unless we pass another parameter indicating that it's running from Kokoro?
Actually, if we pass a param indicating that we are running from Kokoro, we can both move the check for the token and the passing of the $KOKORO... env vars to createcoveragereport.sh.
Sounds good?

Reply:
If we're explicitly passing a --upload or similar flag to createcoveragereport.sh, I think we should assume that we know what we're doing and the environment is already set up correctly. I would rather we kept anything Kokoro-specific in the Kokoro scripts.

I don't know whether I'm being clear in my description - I'm happy to create a PR for what I mean if it would be simpler :)

Reply:
Yes, I see, we are only running coverage if the token is detected anyways.